zetaclient/chains/ton/liteapi/client.go

```diff
@@ -189,6 +191,25 @@ func (c *Client) GetTransactionsSince(
 	return result, nil
 }
 
+// GetTransaction returns account's tx by logicalTime and hash or ErrNotFound.
+func (c *Client) GetTransaction(
+	ctx context.Context,
+	acc ton.AccountID,
+	lt uint64,
+	hash ton.Bits256,
+) (*ton.Transaction, error) {
```

Comment:
Do we need to return a pointer here?

Reply:
https://github.com/zeta-chain/node/pull/3020/commits/9487eb1aa34369df44c15e3603b68009eb6110a5